packages/dumb/src/components/controls/labeled-checkbox/labeled-checkbox.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { FunctionComponent } from 'react'
+import { Checkbox, FormControlLabel } from '@mui/material'
+
+interface LabeledCheckboxProps {
+  checked: boolean
+  onFn: () => void
+  offFn: () => void
+  label: string
+}
+
+export const LabeledCheckbox: FunctionComponent<LabeledCheckboxProps> = (props) => {
+  const { checked, onFn, offFn, label } = props
```

Comment:
What does `onFn` and `offFn` mean?

Reply:
- `onFn`: method called when the checkbox is checked (it's "on")
- `offFn`: method called when the checkbox is unchecked (it's "off")

I thought of some other names but they were very Angular-based (`emitChecked()`, `emitUnchecked()` for instance) but I did not know how to name those in React so I kept the initial names. If you have a better idea, please share it 😄